docs/wiki/CL/cl-clients.md

```diff
@@ -1,178 +1,104 @@
+- [Running a Slasher](https://docs.prylabs.network/docs/prysm-usage/slasher)
+- [Prometheus and Grafana](https://docs.prylabs.network/docs/prysm-usage/monitoring/grafana-dashboard)
+- [Detailed Best Security Practices](https://docs.prylabs.network/docs/security-best-practices)
+
+### Nimbus
+Nimbus, developed in Nim by Status, is optimized for resource efficiency. It supports lightweight devices like smartphones and Raspberry Pi's, conserving resources for other tasks when run on powerful servers. Nimbus features integrated validator client support, remote signing, performance analysis tools, and robust validator monitoring capabilities.
+
+Noteworthy Features:
+- [Run an Execution Client](https://nimbus.guide/eth1.html)
+- [Validator Client](https://nimbus.guide/validator-client.html)
+- [MEV and Builder Integration](https://nimbus.guide/external-block-builder.html)
+- [Prometheus and Grafana](https://nimbus.guide/metrics-pretty-pictures.html)
+
+### Teku
+ConsenSys' Teku is a Java-based Ethereum consensus client offering comprehensive enterprise-grade features. It includes a full beacon node implementation, validator client, REST APIs for node management, Prometheus metrics for monitoring, and external key management for validator signing keys. Teku is suited for enterprise-level Ethereum deployments requiring scalability and operational control.
+
+Noteworthy Features:
+- [Validator Client](https://docs.teku.consensys.io/concepts/proof-of-stake)
+- [Slashing Protection](https://docs.teku.consensys.io/how-to/prevent-slashing/use-a-slashing-protection-file)
+- [Builder and MEV-boost](https://docs.teku.consensys.io/concepts/builder-network)
+- [Detect Doppelganger](https://docs.teku.consensys.io/how-to/prevent-slashing/detect-doppelgangers)
+- [Prometheus and Grafana](https://docs.teku.consensys.io/how-to/monitor/use-metrics)
 
 ### Grandine
+Grandine is a fast and lightweight Ethereum consensus client designed with a focus on high performance and simplicity.
+Developed with parallelization and efficient resource utilization at its core, Grandine aims to push the boundaries of Ethereum's consensus layer by offering a streamlined alternative to existing clients.
+Its architecture is optimized to minimize latency and maximize throughput, making it well-suited for environments where performance is critical.
 
-Originally a proprietary client in Rust, recently became open source
+Noteworthy Features:
+- [Validator Client](https://docs.grandine.io/validator_client.html)
+- [Slashing Protection](https://docs.grandine.io/slashing_protection.html)
+- [Builer and MEV](https://docs.grandine.io/builder_api_and_mev.html)
+- [Running a Slasher](https://github.com/grandinetech/grandine/tree/develop/slasher)
+- [Prometheus](https://docs.grandine.io/metrics.html) and [Grafana](https://github.com/grandinetech/grandine/tree/develop/metrics)
 
-### LambdaClass Client
+### Caplin and LambdaClass
```

Comment:
I would still split these and provide some context. Caplin is basically usable in production, it is integrated in erigon so it is easiest to setup just by enabling a flag. Lambda also deserves it's own paragraph with links to their repo